src/styles/style.css

```diff
@@ -0,0 +1,99 @@
+@import "./stars.css";
+
+html {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
```

Comment:
add padding for all sides for the whole card, and remove margins for all children inside
they should have only margin-bottom to have distance between them and the next element